src/Common/nunit/CommandLineOptions.cs

```diff
@@ -296,6 +298,23 @@ protected virtual void ConfigureOptions()
             this.Add("where=", "Test selection {EXPRESSION} indicating what tests will be run. See description below.",
                 v => WhereClause = RequiredValue(v, "--where"));
 
+            this.Add("params|D=", "Define a test parameter.",
+                v =>
+                {
+                    string parameters = RequiredValue( v, "--params");
+
+                    foreach (string param in parameters.Split(new[] { ';' }))
```

Comment:
Based on #1571, should we be using semi-colons, or is there an easy workaround to that issue?